lib/filter/filter-expr-parser.c

```diff
@@ -47,8 +47,9 @@ static CfgLexerKeyword filter_expr_keywords[] =
   { "!=",                 KW_NUM_NE },
   { ">=",                 KW_NUM_GE },
   { ">",                  KW_NUM_GT },
-  { "level",              KW_LEVEL },
-  { "priority",           KW_LEVEL },
+  { "severity",           KW_SEVERITY },
+  { "level",              KW_SEVERITY },
```

Comment:
shouldn't we use `KWS_OBSOLETE` to indicate that we intend to deprecate level and severity?

Reply:
see above.